ansible/roles/edu-sharing/tasks/jobs.yml

```diff
@@ -1,8 +1,10 @@
 ---
 - name: add replace-mapping-jobs to list of all jobs
   set_fact:
-    edu_jobs_all: "{{ edu_jobs_all | default([], true) + [{'jobClass': 'org.edu_sharing.repository.server.jobs.quartz.BulkEditNodesJob', 'params': {'startFolder': item.startFolder, 'mode': 'ReplaceMapping', 'property': item.property, 'types': 'ccm:io', 'recurseMode': 'All', 'FILE_DATA': item.data}}, {'jobClass': 'org.edu_sharing.repository.server.jobs.quartz.BulkEditNodesJob', 'params': {'startFolder': item.startFolder, 'mode': 'RemoveDuplicates', 'property': item.property, 'types': 'ccm:io', 'recurseMode': 'All'}}] }}"
-  loop: '{{ edu_migration_replace_mapping | default([], true) }}'
+    edu_jobs_all: "{{ edu_jobs_all | default([], true) + [{'jobClass': item.0.jobClass, 'params': item.1 }] }}"
+  with_subelements:
+    - '{{ edu_migration_replace_mapping | default([], true) }}'
+    - parameters
```

Comment:
This task is just for the bulk-edit-replacement-job (see above). Please don't change it. If you want to use a general approach, please use the existing possibility via `edu_jobs_all`. But it would be more user-friendly to create an additional task for your special job, that adds the job-configuration to `edu_jobs_all` (like this task does for bulk-edit-replacement) -> then, the configuration is much easier

Reply:
I rollback this change too